### PR TITLE
Integrated helius action node

### PR DESCRIPTION
**Pull request by abdullahranginwala**

### CODE REVIEW

**Comment by tylim88** [comment]
what version of master branch you working on? 1.0.13?

did you run `yarn setup`?

**Comment by abdullahranginwala** [comment]
I believe axios 1.1.13 was causing some issues. I've fixed that and also the linting issues

**Comment by tylim88** [comment]
image
please click here to sign the CLA

**Comment by abdullahranginwala** [comment]
I've signed the CLA multiple times, I'm not sure why its not updating here
Screenshot 2022-10-28 at 7 26 07 PM

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
         "prettier": "^2.7.1",
         "pretty-quick": "^3.1.3",
         "rimraf": "^3.0.2",
-        "typescript": "^4.8.4"
+        "typescript": "^4.8.4",
+        "axios": "^0.27.2"
```
Comment (HenryHengZJ) [inline package.json:47]: can we remove `axios` here?

**Review by HenryHengZJ — COMMENTED** [review]
The image preferable is svg logo, if svg cant be found, a png logo without the word HELIUS is better

**Review by HenryHengZJ — APPROVED** [review]
Thanks @abdullahranginwala!

**Pull request merged**